README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# Biography
+
+My name is **Kenna Rearick**. I am currently attending the University of Chicago, where I am a **graduate student** in the **Social Service Administration** Advanced Standing Program. I am also a research assistant intern at the University of Chicago **Urban Labs**, where I assist with the Supportive Release Center research study. 
+
+Prior to attending UChicago, I graduated from Youngstown State University with my **Bachelor's in Social Work**. My prior experience includes frontline work with children and families who have experienced homelessness, domestic violence, and other forms of trauma. I am also a proud [AmeriCorps NCCC](https://www.nationalservice.gov/programs/americorps/americorps-programs/americorps-nccc) alum. During my time in AmeriCorps, I traveled through more than 20 states and I lived in the following locations:
+
```

Comment:
good biography but would it be better to rearrange the paragraph into different sections. For example, seperating the link from your body part.

Reply:
Link requirement fulfilled. Trauma work is important! I personally liked how you embedded a link into your paragraph.